File: app/src/main/java/com/example/myapplication/data/model/DogListModel.java
package com.example.myapplication.data.model;

import com.example.myapplication.data.entity.DogListEntity;
import com.example.myapplication.network.dto.DogListDto;

import java.util.ArrayList;
import java.util.List;

public class DogListModel {

    private DogListDto doglistEntity;


    public DogListModel() {
        doglistEntity = new DogListDto();
    }

    public void setDoglistEntity(DogListDto doglistEntity) {
        this.doglistEntity = doglistEntity;
    }

    public DogListDto getDoglistEntity() {
        return doglistEntity;
    }
}

File: app_kotlin/src/main/java/com/example/myapplication/data/model/DogListModel.kt
package com.example.myapplication.data.model

import com.example.myapplication.data.entity.DogListEntity
import com.example.myapplication.network.dto.DogListDto

import java.util.ArrayList

class DogListModel {

    var doglistEntity: DogListDto? = null

    init {
        doglistEntity = DogListDto()
    }
}

